custom/workflow-helper/app-deployer/weak-sock-shop.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   name: carts-db
   namespace: sock-shop
 spec:
+  serviceName: "carts-db"
```

Comment:
Any specific reason for adding this @oumkale . Also, some of these changes are applicable to resilient sock-shop manifest too, I suppose?

Reply:
In resilient one it is already there. 
Request and limits also added